appalerts/templates/appalerts/alert_list.html

```diff
@@ -3,7 +3,7 @@
 {% load static %}
 {% load projectroles_common_tags %}
 
-{% block title %}App Alerts{% endblock title %}
+{% block title %}{% if dismissed %}Dismissed App Alerts{% else %}Active App Alerts{% endif %}{% endblock title %}
```

Comment:
You can simplify this by only having the "Dismissed/Active" part within the if-else block :)

Also, don't forget to also include this in the `<h2>` tag.